android/app/src/main/java/app/organicmaps/intent/Factory.java

```diff
@@ -122,6 +124,15 @@ public boolean process(@NonNull Intent intent, @NonNull MwmActivity target)
             Framework.nativeSetViewportCenter(latlon[0], latlon[1], SEARCH_IN_VIEWPORT_ZOOM);
           }
 
+          return true;
+        }
+        case RequestType.OAUTH2:
+        {
+          SearchEngine.INSTANCE.cancelInteractiveSearch();
```

Comment:
Why the search should be canceled?

Reply:
I don't know if it makes sense. I copied this line from previous `case` blocks.

Shoul be removed?